account.go

```diff
@@ -30,6 +31,32 @@ type Account struct {
 	Email             string   `json:"email"`
 	ACL               []string `json:"acls"`
 }
+type AccountBandwidthBase struct {
```

Comment:
The base types are un-exported and should be lowercase.

Reply:
Fixed: https://github.com/vultr/govultr/pull/356/commits/6ff8a16a1bdb2d230dd3e3637e8aa75beff6ea62